CHANGELOG.md

```diff
@@ -2,11 +2,33 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
-# (TBD)
+# (March TBD, 2021)
+
+**IMPORTANT INSTALL AND UPGRADE NOTES**
+
+Starting in version 18.0, the [AWX Operator](https://github.com/ansible/awx-operator) is the preferred way to install AWX: https://github.com/ansible/awx/blob/devel/INSTALL.md#installing-awx
+
+If you have a pre-existing installation of AWX that utilizes the Docker-based installation method, this install method has ** notably changed** from 17.x to 18.x.  For details, please see:
+
+- https://groups.google.com/g/awx-project/c/47MjWSUQaOc/m/bCjSDn0eBQAJ
+- https://github.com/ansible/awx/blob/devel/tools/docker-compose/docs/data_migration.md
+
```

Comment:
It might be a good idea to also explicitly point to the dev env usage docs, not just the migration notes.  

https://github.com/ansible/awx/blob/devel/tools/docker-compose